telegram/ext/conversationhandler.py

```diff
@@ -143,6 +143,13 @@ class ConversationHandler(Handler[Update]):
             received update and the corresponding ``context`` will be handled by ALL the handler's
             who's :attr:`check_update` method returns :obj:`True` that are in the state
             :attr:`ConversationHandler.TIMEOUT`.
+
+            Note:
+                * `conversation_timeout` with `run_async` may fail, if the :obj:`Promise`
+                  takes longer to finish.
```

Comment:
* :class:`telegram.ext.utils.promise.Promise`
* … longer to finish than than the timeout.

Reply:
👍